images/test-environments/linux/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG CONDA_VER=latest
+ARG UBUNTU_VER=20.04
+ARG OS_TYPE=x86_64
+ARG PY_VER=3.7
+
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -yq curl wget jq
+
+ARG CONDA_VER
+ARG OS_TYPE
+# Install miniconda to /miniconda
+RUN curl -LO "http://repo.continuum.io/miniconda/Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh"
+RUN bash Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh -p /miniconda -b
+RUN rm Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh
+ENV PATH=/miniconda/bin:${PATH}
```

Comment:
Should this be on `PATH` since you're using the shell hooks later to enable conda? As is, this could cause binaries from the base environment to leak through to other environments later.

Reply:
Great catch! I'll remove this line then.